src/main/webapp/app/course/learning-paths/learning-path-management/learning-path-health-status-warning.component.html

```diff
@@ -0,0 +1,9 @@
+<div *ngIf="status" class="card text-bg-warning mb-3">
```

Comment:
What do you think about using [Bootstrap alerts](https://getbootstrap.com/docs/5.3/components/alerts/) here? That would make it look a bit more consistent to the rest of the application.
```suggestion
<div *ngIf="status" class="alert alert-warning mb-3">
```